src/jit/codegencommon.cpp

```diff
@@ -618,6 +618,21 @@ regMaskTP Compiler::compHelperCallKillSet(CorInfoHelpFunc helper)
     }
 }
 
+//------------------------------------------------------------------------
+// compChangeLife: Compare the given "newLife" with last set of live variables and update
+//  codeGen "gcInfo", siScopes, "regSet" with the new variable's homes/liveness.
```

Comment:
grammar suggestion: "the new variable's homes/liveness." => "the new variable homes/liveness."

Reply:
I take it, thanks!